### PR TITLE
#53(admins) make messages table component be similar to the users table component

### PR DESCRIPTION
**Pull request by MaxiFrancis**

### CODE REVIEW

**Comment by MaxiFrancis** [comment]
https://github.com/user-attachments/assets/05b128fd-e8bd-4200-93e0-e87b92c569ea
On save it shows the rows but just for a bit.

**Comment by ghefraim** [comment]
image

File: src/components/admins/MessagesTable.jsx [hunk 1]
```diff
@@ -1,60 +1,287 @@
...
+        isFilterable: true,
+      },
+      {
+        Header: "Citit",
+        accessor: "isRead",
+        Cell: ({ value }) => (value ? "Citit" : "Necitit"),
+        isFilterable: true,
+      },
+      {
+        Header: "More",
```
Comment (ghefraim) [inline src/components/admins/MessagesTable.jsx:124]: I think we can remove this "More" column. There are no extra details to be shown here

**Review by ghefraim — CHANGES_REQUESTED** [review]
Pls remove the "More" functionality from the Messages table, and pls fix the data not being shown

**Pull request closed without merging**